build.sh

```diff
@@ -17,44 +17,40 @@ set -ex
 cd "$(dirname "${BASH_SOURCE[0]}")"
 
 #type="Release"
-#generator="Unix Makefiles"
+generator="Unix Makefiles"
 
 type=${type:-"Debug"}
-generator=${generator:-"Ninja"}
+# generator=${generator:-"Ninja"}
```

Comment:
Switch these back before merging.  If you want to use Makefiles, you can just do it this way: `generator="Unix Makefiles" ./build.sh` without having to change the file itself.